testsuite/features/secondary/srv_manage_channels_page.feature

```diff
@@ -44,21 +44,21 @@ Feature: Managing channels
   Scenario: Fail when trying to use reserved names for channels
     When I follow the left menu "Software > Manage > Channels"
     And I follow "Create Channel"
-    And I enter "SLE-12-Cloud-Compute5-Pool for x86_64" as "Channel Name"
+    And I enter "openSUSE-Leap-15.5-Pool for x86_64" as "Channel Name"
```

Comment:
We need to double check if we need to mirror this product, in order to have it shown in the product list, and then have this test passing.

Reply:
![image](https://github.com/user-attachments/assets/b2467135-d4e0-49b7-be47-dd8d3af7942c)

Checked. All good.